db/schema.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_214_221_701) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'budgets', force: :cascade do |t|
+    t.string 'name'
+    t.decimal 'amount'
+    t.integer 'user_id'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
+
+  create_table 'budgets_groups', force: :cascade do |t|
+    t.bigint 'group_id', null: false
+    t.bigint 'budget_id', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['budget_id'], name: 'index_budgets_groups_on_budget_id'
+    t.index ['group_id'], name: 'index_budgets_groups_on_group_id'
+  end
+
+  create_table 'groups', force: :cascade do |t|
+    t.string 'name'
+    t.string 'icon'
+    t.integer 'user_id'
```

Comment:
- [ ] Kindly make sure that all foreign keys have indexes as stated in the requirements.

Reply:
Done!